### ISSUE TITLE
Question submission behaviour on hitting enter key

### ISSUE THREAD

**Issue by WildRyc** [KingSupernova31/RulesGuru#199]
![image](https://github.com/KingSupernova31/RulesGuru/assets/4632594/a471adb1-923f-4e8d-b8de-8248c0e3ec61)

Hitting the enter key while submitting a question triggers the submit button. Is this the desired behaviour?

**Comment by KingSupernova31** [comment]
Yes; most questions don't need line breaks, and it makes submission a lot faster to let people do it with a keypress rather than having to use the mouse.

If you hold shift while pressing enter it will input a line break. I agree it's not great that this is a hidden feature, since as you point out, many people will assume that enter will input a line break. I'd love to hear ideas for improvement!

**Comment by WildRyc** [comment]
Option 1) Indicate on the page what the behaviour currently is.
2) Toggle as a user setting
3) Just require the submit button be used to process, removing this behaviour entirely.

**Comment by KingSupernova31** [comment]
Added an indication on the page
